gimme_aws_creds/okta.py

```diff
@@ -29,6 +29,10 @@
 from gimme_aws_creds.webauthn import WebAuthnClient, FakeAssertion
 from . import errors, ui, version
 
+from aws_okta_keyman import duo
```

Comment:
I'd really prefer not to use a command-line tool as a library, particularly when they are not expecting it.  Further, placing the whole project as a dependency for just the duo capability feels unnecessary.

Lets pull just the `Duo` class into this project, and we'll be sure to leave the license and copyright data at the top:
https://tldrlegal.com/license/apache-license-2.0-(apache-2.0)

However, lets also add a link to his repository right below the copyright:
https://github.com/nathan-v/aws_okta_keyman

In the mean time, I'll do some further checking to ensure this is all kosher.

Reply:
Done